hiredis/hiredis.pyi

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Callable, Optional, Union
+
+class HiredisError(Exception): ...
+class ProtocolError(HiredisError): ...
+class ReplyError(HiredisError): ...
+
+class Reader:
+    def __init__(
+        self,
+        protocolError: Callable[[str], Exception] = ...,
+        replyError: Callable[[str], Exception] = ...,
+        encoding: Optional[str] = ...,
+        errors: Optional[str] = ...,
+    ) -> None: ...
+    def feed(
+        self, __buf: Union[str, bytes], __off: int = ..., __len: int = ...
+    ) -> None: ...
+    def gets(self, __shouldDecode: bool = ...) -> Any: ...
+    def setmaxbuf(self, __maxbuf: Optional[int]) -> None: ...
+    def getmaxbuf(self) -> int: ...
+    def len(self) -> int: ...
```

Comment:
future PR, but might be useful to setup `mypy` or one of the other type checkers to help catch any potential type issues by type checking the `test` module